src/matchmaking/output/output_fastchess.hpp

```diff
@@ -8,18 +8,21 @@ namespace fast_chess {
 
 class Fastchess : public IOutput {
    public:
-    void printInterval(const SPRT& sprt, const Stats& stats, const std::string& first,
+        Fastchess(bool report_penta = true) : report_penta_(report_penta) {}
+
+        void printInterval(const SPRT& sprt, const Stats& stats, const std::string& first,
                        const std::string& second, int current_game_count) override {
         std::cout << "--------------------------------------------------\n";
         printElo(stats, first, second, current_game_count);
         printSprt(sprt, stats);
-        printPenta(stats);
+        if (report_penta_ == true){printPenta(stats);}
         std::cout << "--------------------------------------------------\n";
     };
 
     void printElo(const Stats& stats, const std::string& first, const std::string& second,
                   std::size_t current_game_count) override {
-        const Elo elo(stats.penta_WW, stats.penta_WD, stats.penta_WL, stats.penta_DD,
+        const Elo elo(stats.wins, stats.losses, stats.draws);
```

Comment:
move it into the if scope?

Reply:
since its a declaration then if i move it into the if scope wouldnt the variable be limited to the if scope only? idk how to declare it separately outside the if scope

Reply:
nvm, fixed with https://github.com/Disservin/fast-chess/pull/229/commits/669363f34b42f6a061e44a1477861353ee5b1c68